frontend/dockerfile/instructions/commands_runmount.go

```diff
@@ -108,6 +108,9 @@ type Mount struct {
 	CacheID      string
 	CacheSharing string
 	Required     bool
+	Mode         *uint64
+	UID          *uint64
+	GID          *uint64
```

Comment:
Just dropping by, but wondering: should these be `*uint32` instead of `*uint64` ? (I think [Linux uses 32 bit for this](http://www.linfo.org/uid.html))

In moby, looks like we're using `int` in some places (which may not be correct); https://github.com/moby/moby/blob/b3e9f7b13b0f0c414fa6253e1f17a86b2cff68b5/pkg/idtools/idtools.go#L105-L110

Reply:
This is just to match the return type from parse. They are casted before actual use. I'll change the bitsize of the parse though.

Reply:
Ah, thx, clear 👍